### DISCUSSION TITLE
Unable to print `b:T` and `p:T` variables in a script

### DISCUSSION THREAD

**Discussion by elcponce** [gopro/labs#1073] [Q&A] [answered]
Hi everyone,

I’m learning some of the scripting features in the Labs firmware and encountered an issue. I can’t seem to print the battery temperature (b:T) or the processor temperature (p:T) using the following code:

`=Ab:T=Cp:T"Battery Level: $b[%]""Battery Temperature: $A[C]""Processor Temperature $C[C]"!2N!R`

It looks like `$A` is storing the battery level `b` and $C is holding the sound pressure level `p`, but the `:T` part of the variable assignments seems to be ignored. I couldn’t find any examples of using these variables with the `:T` section in the repository.

Has anyone worked with these types of variables before, especially with the : character?

Thanks in advance!

**Comment by dnewman-gpsw** [comment]
Works on HERO13 here.

**Reply by elcponce** [reply]
Hmmmm, I'm using a HERO11 Mini to run the script. Maybe these variables aren't available in this model? 🤔 🤔

**Reply by dnewman-gpsw** [reply, marked as answer]
Correct, those new variables where just recently added to HERO13.

**Comment by dnewman-gpsw** [comment]
https://github.com/user-attachments/assets/73e553c1-687a-4b1c-8af0-2ba9ec753b0b
